setup.py

```diff
@@ -4,7 +4,7 @@
 
 from setuptools import setup
 setup(name='perfact-zodbsync',
-      version='3.10.0',
+      version='3.10.1',
```

Comment:
Since this version ships with DB-Utils 3.11, I would like to set the version accordingly.
```suggestion
      version='3.11.0',
```